javascript/app.js

```diff
@@ -183,6 +202,11 @@ cocktailApp.revealButton.addEventListener("click", function () {
                 // <li><span class="measurements">2 oz</span> Rum</li>
                 ingredientList.appendChild(listElement);
             })
+            setTimeout(function() {
```

Comment:
This is so that the recipe only loads in after the buttons fade out (so there's no hard jump of the content moving up the page)